code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import backArrow from 'arrow.svg'
+import { Link } from 'react-router-dom'
+
+export const MovieDetails = () => {
+
+  const { id } = useParams()
+  const [movie, setMovie] = useState([])
+
+  useEffect(() => {
+
+    fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=db37f43a686cbd6ad83202f3d97ecfb5&language=en-US&page=1`)
+      .then((res) => res.json())
+      .then((json) => {
+        setMovie(json)
+        console.log(json.overview)
+        console.log(json)
+      })
+
+  }, [id])
+
+  const bgLink = `https://image.tmdb.org/t/p/w1280${movie.backdrop_path}`
+  const bgImage = {
+    backgroundImage: `url(${bgLink})`
+  }
+
+  return (
+    <div>
+      <div style={bgImage}>
```

Comment:
you could also have put `{backgroundImage: `url(${bgLink})`}` here.
But nothing wrong to split it up!